CountOnMe/Model/Operations+Extension.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  CalculatorModel+Extension.swift
+//  CountOnMe
+//
+//  Created by Redouane on 08/03/2024.
+//  Copyright © 2024 Vincent Saluzzo. All rights reserved.
+//
+
+import Foundation
+
+extension CalculatorModel {
+
+    enum MathOperator {
```

Comment:
**suggestion (non-blocking)**: With all the things going in this enum, it can be in its own file.